docs/getting-started/integrations-cdn/vuejs-v2.md

```diff
@@ -0,0 +1,147 @@
+---
+menu-title: Vue.js 2.x
+meta-title: Vue.js 2.x rich text editor component with CDN | CKEditor 5 documentation
+meta-description: Install, integrate and configure CKEditor 5 using the Vue.js 2.x component with CDN.
+category: cloud
+order: 40
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# Vue.js 2.x rich text editor component
+
+<p>
+	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-vue2" target="_blank" rel="noopener">
+		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-vue2.svg" alt="npm version" loading="lazy">
+	</a>
+</p>
+
+<info-box warning>
+	This guide is about the CKEditor&nbsp;5 integration with Vue.js 2.x. However, Vue 2 has reached EOL and is no longer actively maintained. To learn more about the integration with Vue.js 3+, check out the {@link getting-started/integrations-cdn/vuejs-v3"Rich text editor component for Vue.js 3+} guide.
+</info-box>
+
+Vue.js is a versatile framework for building web user interfaces. CKEditor&nbsp;5 provides an official Vue component you can use in your application. This guide assumes that you already have a Vue 2 project.
```

Comment:
```suggestion
CKEditor 5 has an official Vue 2 integration that you can use to add a rich text editor to your application. This guide will help you install it and configure to use the CDN distribution of the CKEditor 5.
```

I think that we don't need to tell devs what Vue is. If they are on this page, then they very likely already know.

Reply:
Yeah, you're right. But I'm a little hesitant about using the "official integration" phrase in this case.